### PR TITLE
Update DocsSidebar and DocsNav

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
* update sidebar trigger UI indicator so that it's themeable
* docs nav: update active state color, add hover state CSS
* add `forceExpand` prop to DocsNav
* clean up `DocsTemplatePlain` css -- remove commented code
* update `icon-figma.svg` asset
* remove `grid-row-gap` from DocsPageHeader
* updates component page links spacing

### CODE REVIEW

**Review by edburyenegren-okta — APPROVED** [review]
Sass changes are 👍

File: packages/vuepress-theme-odyssey/components/DocsNav.vue [hunk 1]
```diff
@@ -88,8 +88,12 @@ export default {
       type: Array, // TODO: figure out if theres a way to better 'type' this with vue
       required: true
     },
+    forceExpand: {
+      type: Boolean,
+      default: false
+    },
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/components/DocsNav.vue:94]: Shouldn't `isSubmenuExpanded` already handle this?

**Review by jmelberg-okta — APPROVED** [review]
Changes seem fine to me!

**Comment by jmelberg-okta** [comment]
Please make sure to squash and merge!

File: packages/vuepress-theme-odyssey/components/DocsNav.vue (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/components/DocsNav.vue:94]: `isSubmenuExpanded` deals with whether or not the submenu is visible (or expanded)

Perhaps a clearer name would be `isSubmenuVisible`? Im open to suggestions here, its not as clear as I would like.

**Pull request merged**